### ISSUE TITLE
Totem Crafting

### ISSUE THREAD

**Issue by Ormatist** [The-Iceburg/TotemsPlus#39]
Add the ability for custom crafting to be enabled for each totem this means we would have to code the custom recipes and the ability for the user to create these themselves, this would involve a 3x3 gird where the user can select each item that is currently obtainable in Minecraft support would then have to be provided whenever the game is updated to include more items.

**Comment by Ormatist** [comment]
There should also be an option to be able to craft the original totem

**Comment by Dockuin** [comment]
May be more difficult than first thought
Custom Crafting can only be done with normal items without nbt data as far as I'm aware - meaning you would only be able to craft the original totem - for full implementation, see here: [Custom Crafting](https://datapackcenter.com/projects/custom-crafting-tutorial.70/)
Will look into further.

**Comment by Ormatist** [comment]
Yes after looking into it I can see where your coming from. I'll have to experiment with it a bit when I return from holiday. I've seen it possible in vids such as this --> https://youtu.be/cf1NAHT1Upo but wether or not this is a mod or datapack I'm unsure. If not other datapacks I've seen just take advantage of a custom tile block e.g. a totem crafting table so maybe this would be a better route to take

**Comment by Dockuin** [comment]
Custom tile entity might be the way to go anyway if we wanted to kill 2 birds with one stone considering the totems villager idea

Issue #11

**Comment by Ormatist** [comment]
That's prbbly a better idea, if not this is always possible: https://youtu.be/Pt7y6PNP6KY
